src/languages/ruby.md

```diff
@@ -4,8 +4,8 @@ Platform.sh supports deploying any Ruby application. Your application can use
 any Ruby application server such as Unicorn or Puma and deploying a Rails or
 a Sinatra app is very straight forward.
 
-### Supported versions
-#### Ruby MRI
+## Supported versions
```

Comment:
Ori specifically pushed to say MRI here.  I don't know Ruby well enough to know if that's sufficiently relevant to call out or not.